src/ussd_registration.js

```diff
@@ -124,6 +124,7 @@ go.app = function() {
     // START STATE
 
         self.add('state_start', function(name) {
+            self.im.user.answers = {};
```

Comment:
We need a test to see that this is working. There's an example test in go-ndoh nurse_ussd.